cmd/clusterctl/validation/validate_cluster_api_objects.go

```diff
@@ -43,7 +43,7 @@ func ValidateClusterAPIObjects(ctx context.Context, w io.Writer, c client.Client
 	}
 
 	machines := &clusterv1alpha1.MachineList{}
-	if err := c.List(ctx, client.InNamespace(namespace), machines); err != nil {
+	if err := c.List(ctx, machines, client.InNamespace(namespace)); err != nil {
```

Comment:
What caused the ordering to change here? Is it just related to the updated client generation?

Reply:
It's from a controller-runtime change to the method signature

Reply:
https://github.com/kubernetes-sigs/controller-runtime/pull/294

Reply:
we changed the method signature to be in line with other methods and to support easier option passing.